lib/rspec/core/project_initializer/spec/spec_helper.rb

```diff
@@ -59,7 +59,7 @@
   # recommended. For more details, see:
   #   - http://myronmars.to/n/dev-blog/2012/06/rspecs-new-expectation-syntax
```

Comment:
Can you update this link as well?  It redirects to http://rspec.info/blog/2012/06/rspecs-new-expectation-syntax/